Description: Add testing script
 "testing.sh" is added as take-home-test required.
 .
 hello (2.11-0ubuntu1) focal; urgency=medium
 .
   * Initial release
Author: Qian-Hao Huang <qiao.helloworld@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2023-04-23

--- /dev/null
+++ hello-2.11/testing.sh
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo "this is a test from Qian-Hao Huang" >&2
